include/grpc/impl/channel_arg_names.h

```diff
@@ -268,12 +281,12 @@
    using the global instance, which is not the intended way to use xDS.
    Currently, this will (a) add unnecessary load on the xDS server and
    (b) break use of CSDS, and there may be additional side effects in
-   the future. */
+   the future. A string value. */
```

Comment:
no need to change this, the description already makes it clear that it is a string